micro-deps/micro-deps-spring-config/src/main/java/com/ofg/infrastructure/discovery/SpringCloudZookeeperDiscoveryClientConfiguration.groovy

```diff
@@ -1,56 +0,0 @@
-package com.ofg.infrastructure.discovery
-
-import org.apache.curator.framework.CuratorFramework
-import org.apache.curator.x.discovery.details.InstanceSerializer
-import org.apache.curator.x.discovery.details.JsonInstanceSerializer
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean
-import org.springframework.cloud.commons.util.InetUtils
-import org.springframework.cloud.zookeeper.discovery.ZookeeperDiscoveryClient
-import org.springframework.cloud.zookeeper.discovery.ZookeeperDiscoveryClientConfiguration
-import org.springframework.cloud.zookeeper.discovery.ZookeeperDiscoveryProperties
-import org.springframework.cloud.zookeeper.discovery.ZookeeperInstance
-import org.springframework.cloud.zookeeper.discovery.ZookeeperServiceDiscovery
-import org.springframework.cloud.zookeeper.discovery.dependency.ZookeeperDependencies
-import org.springframework.context.annotation.Bean
-import org.springframework.context.annotation.Primary
-import org.springframework.context.annotation.Profile
-
-import static com.ofg.config.BasicProfiles.SPRING_CLOUD
-/**
- * Custom configuration that helps initialize deprecated ZookeeperServiceDiscovery and ZookeeperDiscoveryProperties
- */
-@Profile(SPRING_CLOUD)
-class SpringCloudZookeeperDiscoveryClientConfiguration extends ZookeeperDiscoveryClientConfiguration {
```

Comment:
This configuration was prepared after upgrading Spring Cloud from Camden.SR7 to Dalston.SR4 to support creation of deprecated `ZookeeperServiceDiscovery`... It was replaced with Spring Cloud stuff: `ZookeeperServiceRegistry` and appropriate auto configurations: `ZookeeperDiscoveryClientConfiguration`, `ZookeeperDiscoveryAutoConfiguration`, `ZookeeperServiceRegistryAutoConfiguration`, Z`ookeeperAutoServiceRegistrationAutoConfiguration`. See `ZookeeperServiceResolverConfiguration`.